### ISSUE TITLE
Now that i have updated to 0.15,   i get this error with bevy_asset_loader 

### ISSUE THREAD

**Issue by ethereumdegen** [AmionSky/bevy_obj#20]
This crate used to work perfectly for me with bevy_asset_loader now it is horrible.  Now i keep getting this error :

```
thread 'main' panicked at /home/andy/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_asset-0.15.0/src/handle.rs:357:13:
The target Handle<bevy_mesh::mesh::Mesh>'s TypeId does not match the TypeId of this UntypedHandle
```

I think it may be because it is assuming the type as Scene for some of them instead of Mesh ?  But afaik, i cannot tell it to load as a Mesh like your example because I am using bevy_asset_loader like this: 

```

#[derive(AssetCollection, Resource)]
pub struct MeshAssets {
   
     #[asset(path = "../artifacts/game_assets/models/meshes", collection(typed, mapped))]
    pub(crate) meshes: HashMap<AssetFileName, Handle<Mesh>>,


}
```

hmm. I really need a solution here.  Can i make it work like it used to in 0.14 ?  Can i enforce that they are all Mesh and not Scene?    Can i somehow configure the bevy_asset_loader differently?  What do you think . thanks

**Comment by AmionSky** [comment]
I have never used bevy_asset_loader so I'm unsure what its doing. But you can select which mode you want via features.

```toml
[dependencies.bevy_obj]
default-features = false
features = ["mesh"]
```

**Comment by ethereumdegen** [comment]
oh lol i think this is the issue 


[dependencies.bevy]
version = "0.15.0-rc"

**Comment by AmionSky** [comment]
"0.15.0-rc" resolves to "0.15.0". So a 'cargo update' should fix it
